addons/skin.estuary/xml/DialogPVRInfo.xml

```diff
@@ -24,6 +24,17 @@
 				<height>461</height>
 				<aspectratio>keep</aspectratio>
 				<texture fallback="DefaultTVShows.png">$INFO[ListItem.Icon]</texture>
+				<visible>!String.IsEmpty(ListItem.EpgEventIcon)</visible>
+			</control>
+			<control type="image">
+				<include>OpenClose_Left</include>
+				<left>114</left>
+				<top>236</top>
+				<width>461</width>
+				<height>461</height>
+				<aspectratio>keep</aspectratio>
+				<texture>$INFO[ListItem.EpgEventIcon]</texture>
+				<visible>String.IsEmpty(ListItem.EpgEventIcon)</visible>
```

Comment:
This makes zero sense as you state here that you want to show the epg event image only if its empty.